apache/config.sls

```diff
@@ -29,112 +80,39 @@ include:
       - module: apache-reload
       - service: apache
     - context:
-      apache: {{ apache | json }}
+        apache: {{ apache | json}}
+        server_config: {{ server_config | json }}
 
 {{ apache.vhostdir }}:
   file.directory:
     - makedirs: True
     - require:
       - pkg: apache
+    - user: root
+    - group: root
+    - dir_mode: 755
+    - file_mode: 644
+    - recurse:
+      - user
+      - group
+      - mode
     - watch_in:
       - module: apache-restart
     - require_in:
       - module: apache-restart
       - module: apache-reload
       - service: apache
 
-{% if grains['os_family']=="Debian" %}
-/etc/apache2/envvars:
-  file.managed:
-    - template: jinja
-    - source:
-      - salt://apache/files/{{ salt['grains.get']('os_family') }}/envvars-{{ apache.version }}.jinja
-    - require:
-      - pkg: apache
-    - watch_in:
-      - module: apache-restart
-    - require_in:
-      - module: apache-restart
-      - module: apache-reload
-      - service: apache
-
-{{ apache.portsfile }}:
-  file.managed:
-    - template: jinja
-    - source:
-      - salt://apache/files/{{ salt['grains.get']('os_family') }}/ports-{{ apache.version }}.conf.jinja
-    - require:
-      - pkg: apache
-    - watch_in:
-      - module: apache-restart
-    - require_in:
-      - module: apache-restart
-      - module: apache-reload
-      - service: apache
-    - context:
-      apache: {{ apache | json }}
-
-{% endif %}
-
-{% if grains['os_family']=="RedHat" %}
-{{ apache.confdir }}/welcome.conf:
-  file.absent:
-    - require:
-      - pkg: apache
-    - watch_in:
-      - module: apache-restart
-    - require_in:
-      - module: apache-restart
-      - module: apache-reload
-      - service: apache
-{% endif %}
-
-{% if grains['os_family']=="Suse" or salt['grains.get']('os') == 'SUSE' %}
-/etc/apache2/global.conf:
-  file.managed:
-    - template: jinja
-    - source:
-      - salt://apache/files/{{ salt['grains.get']('os_family') }}/global.config.jinja
-    - require:
-      - pkg: apache
-    - watch_in:
-      - module: apache-restart
-    - require_in:
-      - module: apache-restart
-      - module: apache-reload
-      - service: apache
-    - context:
-      apache: {{ apache | json }}
-{% endif %}
-
-{% if grains['os_family']=="FreeBSD" %}
-/usr/local/etc/{{ apache.service }}/envvars.d/by_salt.env:
-  file.managed:
-    - template: jinja
-    - source:
-      - salt://apache/files/{{ salt['grains.get']('os_family') }}/envvars-{{ apache.version }}.jinja
-    - require:
-      - pkg: apache
-    - watch_in:
-      - module: apache-restart
-    - require_in:
-      - module: apache-restart
-      - module: apache-reload
-      - service: apache
 
-{{ apache.portsfile }}:
+/etc/httpd/conf.d/welcome.conf:
```

Comment:
This hardcoded path won't fly with Debian(ish) or FreeBSD.

Reply:
yes done ! didn't saw it :) thanks